client-sdk/ts-web/core/src/types.ts

```diff
@@ -1208,6 +1478,12 @@ export interface RegistryUnfreezeNode {
     node_id: Uint8Array;
 }
 
+/**
+ * ValidatorSetConstraint specifies that the entity must have a node that is part of the validator
+ * set. No other options can currently be specified.
+ */
+export type RegistryValidatorSetConstraint = Map<never, never>;
```

Comment:
The constraints types should probably be grouped together? Now it seems that some are 300 lines higher.

Reply:
Also why does it have this `Registry` prefix that the original type does not have?

Reply:
I went with alphabetical sorting in types.ts ):

these types almost all have a prefix for what Go package they're from